src/api/basic-subscriptions.js

```diff
@@ -99,6 +99,7 @@ export const AutoPromptType = {
   NONE: 'none',
   CONTRIBUTION: 'contribution',
   CONTRIBUTION_LARGE: 'contribution_large',
+  METER_LARGE: 'meter_large',
```

Comment:
These are params that are provided on the page to indicate the flow (contribution vs subscription). I don't think this belongs as a "type" here since it is actually a modal that only comes up under a subscription flow

Reply:
Synced offline and we believe the best approach is to reuse MeterToastApi for displaying the new meteriframe. For the entry point that decides which prompt to show, Ricky is working on a PR that the AutoPromptManager will be responsible for that.